docs/docs/config/encoders.md

```diff
@@ -29,13 +29,49 @@ If `CONFIG_EC11` is enabled, exactly one of the following options must be set to
 
 ### Devicetree
 
+#### Keymap Sensor Config
+
+For shields/boards that export a `sensors` node configuration label, both global and per-sensor settings can be set by overriding the properties there.
+
+To override the general settings, update them on the exported `sensors` node, e.g.:
+
+```
+&sensors {
+    triggers-per-rotation = <18>;
+};
+```
+
+Per sensor overrides can be added with ordered nested nodes with the correct overrides, e.g.:
```

Comment:
Might be useful to note that these are listed in the order that the sensors are defined in the board/shield, and the names don't matter.

Reply:
Added a note about this.